Automattic/newspack-plugin
# fix(ras-acc): update transactional email fonts to sans serif

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1207817176293825/1207990881852220/f

This PR updates the font used for transactional emails to Arial (fallback default sans-serif) to be closer to the designs.

![Screenshot 2024-08-08 at 16 37 25](https://github.com/user-attachments/assets/3e81df3d-b29c-45f8-88a6-a1176dca5c0f)

This PR also sneaks in a small fix where the cancellation button text color was not correct in the editor:

![Screenshot 2024-08-08 at 16 56 04](https://github.com/user-attachments/assets/ec336783-cd34-441d-82e0-a3f191c4f236)

### How to test the changes in this Pull Request:

1. As admin, go to Newspack > Engagement, select the Show Advanced Settings button, then scroll down to the transactional emails section.
2. Reset each of the emails
3. Do the same for reader revenue emails (Newspack > Reader Revenue > Emails
4. View each of the emails in the post editor:
  - Confirm the editor contains Arial (or whatever default sans-serif is) font.
  - Send a test email via the post editor settings and confirm the emails contain Arial font. 

### Other information:
